contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,26 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+RUN dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony; dnf -y clean all
+RUN dnf -y install skopeo buildah; dnf -y clean all
+RUN systemctl enable podman.service
+RUN printf "\nconfdir /etc/chrony.d\n" >> /etc/chrony.conf
+
+COPY 10-inotify-instances.conf /usr/etc/sysctl.d/10-inotify-instances.conf
```

Comment:
The preference here is is actually to use `/usr/lib/sysctl.d`.  This configuration is part of the OS.

Otherwise, use `/etc`.  Don't use `/usr/etc`.

Reply:
Fixed.